### PR TITLE
Add organization to ApiTracking embedded

### PR DESCRIPTION
**Pull request by ashkarpetin**
## Summary

Add organization `_embedded` to ApiTracking

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

File: openapi/components/schemas/Tracking/ApiTracking.yaml [hunk 1]
```diff
@@ -89,6 +89,19 @@ properties:
         description: Specifies if the API request is completed by a Rebilly Support Team member.
         type: boolean
         x-sortable: true
+  organization:
+    type: object
+    description: Organization of the user who made the request.
```
Comment (VladimirBerdnik) [inline openapi/components/schemas/Tracking/ApiTracking.yaml:94]: user may have many organizations so the description is not quite correct

```suggestion
    description: Organization in scope of which the request was made.
```

The suggestion is also difficult to read, but demonstrates the idea

Comment (swilczynski) [reply openapi/components/schemas/Tracking/ApiTracking.yaml:94]: How about `Organization from which the request is made.`

Comment (ashkarpetin) [inline openapi/components/schemas/Tracking/ApiTracking.yaml:94]:
```suggestion
    description: Organization from which the request is made.
```

**Comment by ashkarpetin** [comment]
js please review

**Review by swilczynski — APPROVED** [review]

**Review by vykhrystiuk — APPROVED** [review]

**Review by noon-dawg — APPROVED** [review]

**Review by adamaltman — APPROVED** [review]

**Pull request merged**